docs/src/main/sphinx/release/release-383.md

```diff
@@ -0,0 +1,125 @@
+# Release 383 (1 June 2022)
+
+## General
+
+* Introduce `json_exists`, `json_query`, and `json_value` [JSON functions](/functions/json). ({issue}`9081`)
+* Add AWS IAM role support for exchange spooling on S3. ({issue}`12444`)
+* Improve query performance by reducing worker-to-worker communication overhead. ({issue}`11289`)
+* Improve performance and reduce memory usage of queries that contain aggregations. ({issue}`12336`)
+* Improve performance of correlated queries involving distinct aggregations. ({issue}`12564`)
+
+## Security
+
+* Add access control for functions based on function kind.
+
+## Web UI
+
+* Clarify format of cumulative user memory on query details page.
+
+## Accumulo connector
+
+* Fail creating a new table if a table comment is specified. Previously, the
+  comment was ignored. ({issue}`12452`)
+
+## BigQuery connector
+
+* Fail creating a new table if a table comment or a column comment is specified.
+  Previously, the comment was ignored. ({issue}`12452`, {issue}`12574`)
+
+## Cassandra connector
+
+* Fail creating a new table if a table comment or a column comment is specified.
+  Previously, the comment was ignored. ({issue}`12452`, {issue}`12574`)
+
+## ClickHouse connector
+
+* Fix incorrect query results for certain aggregation queries where aggregation
+  is pushed down  to be executed by the remote database, but the aggregation
+  function result is not needed to evaluate the query. ({issue}`12598`)
+
+## Delta Lake connector
+
+* Support storing a table comment when creating a new table. ({issue}`12452`)
+* Fix incorrect `table already exists` error caused by a client timeout when
+  creating a new table. ({issue}`12300`)
+* Fail creating a new table if a column comment is specified. Previously, the
+  comment was ignored. ({issue}`12574`)
+
+## Iceberg connector
+
+* Enable execution of the `optimize` procedure for Iceberg spec v2 tables. ({issue}`12351`)
+* Rename `hive.target-max-file-size` to `iceberg.target-max-file-size` and
+  `hive.delete-schema-locations-fallback` to `iceberg.delete-schema-locations-fallback`. ({issue}`12330`)
+
+## Kudu connector
+
+* Fail creating a new table if a table comment or a column comment is specified.
+  Previously, the comment was ignored. ({issue}`12452`, {issue}`12574`)
+
+## MariaDB connector
+
+* Fix incorrect query results for certain aggregation queries where aggregation
+  is pushed down  to be executed by the remote database, but the aggregation
+  function result is not needed to evaluate the query. ({issue}`12598`)
+* Fail creating a new table if a column comment is specified. Previously, the
+  comment was ignored. ({issue}`12574`)
+
+## Memory connector
+
+* Fail creating a new table if a table comment or a column comment is specified.
```

Comment:
Wrong issue? 12452 is about Delta Lake, not memory connector, according to its title.

Reply:
I don't think so - 71dcb669621bce7440af26dd94bfcd780d2c680b included this change, and was a part of that PR.